remaining packer content (#13285)
hashicorp/packer: website/content/docs/post-processors/checksum.mdx

---
description: >

>   The `checksum` post-processor computes the checksum for the artifact list
>   from an upstream builder or post-processor so that you can verify artifacts. 

Reviewer: ```suggestion
  The `checksum` post-processor computes the checksum for the artifact, letting you verify it.
```
